simplekv/db/sql.py

```diff
@@ -44,6 +45,18 @@ def _get(self, key):
     def _open(self, key):
         return BytesIO(self._get(key))
 
+    def _move(self, source, dest):
+        if not self._has_key(source):
+            raise KeyError(source)
+        con = self.bind.connect()
+        with con.begin():
+            # delete the potential existing previous key
+            con.execute(self.table.delete(self.table.c.key == dest))
+            con.execute(self.table.update().values(key=dest).where(self.table.c.key == source))
```

Comment:
I am pretty weak on SQLAlchemy syntax and semantics. Would you mind elaboration what is supposed to happen here?